### PR TITLE
Caramel: Version 1.010 added

### PR DESCRIPTION
**Pull request by vv-monsalve**
2f73c81: [gftools-packager] Caramel: Version 1.010 added

* Caramel Version 1.010 taken from the upstream repo https://github.com/googlefonts/caramel at commit https://github.com/googlefonts/caramel/commit/bd1d693ea8d578604a2ea2411672984619938faa.

 57d5601: [gftools-packager] ofl/caramel remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by vv-monsalve** [comment]
This is part of the Batch2 of TypeSetIt projects.

> 🔥 FAIL: Does DESCRIPTION file contain broken links?
> 🔥 FAIL: Does METADATA.pb copyright field contain broken links?

I don't find any broken links

> 🔥 FAIL Designer Robert Leuschke still does not have a webpage link on the catalog. Please provide one. [code: missing-link]

It was already done in #3463

**Comment by RosaWagner** [comment]
@vv-monsalve It indicates as broken because the repo is still private

File: ofl/caramel/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: "Caramel"
+designer: "Robert Leuschke"
+license: "OFL"
+category: "DISPLAY"
```
Comment (RosaWagner) [inline ofl/caramel/METADATA.pb:4]: HANDWRITING would fit better no? Playball has this issue too (is the only one as far as I see)

**Comment by RosaWagner** [comment]
Otherwise LGTM, we can merge as soon as the category is confirmed or corrected

**Comment by vv-monsalve** [comment]
Updated

## Caramel: Version 1.010 added

---
 375b262: [gftools-packager] Caramel: Version 1.010 added

* Caramel Version 1.010 taken from the upstream repo https://github.com/googlefonts/caramel at commit https://github.com/googlefonts/caramel/commit/497341c437891ac272f8b3ac68b721f07031f3d3.

 55f81cc: [gftools-packager] ofl/caramel remove METADATA "source".  google/fonts#2587

File: ofl/caramel/METADATA.pb (hunk 1 above)
Comment (vv-monsalve) [reply ofl/caramel/METADATA.pb]: > HANDWRITING would fit better no? Playball has this issue too (is the only one as far as I see)

Oh yes, thanks! Done.

**Comment by davelab6** [comment]
Repo is public

**Pull request merged**